runtime/doc/eval.txt

```diff
@@ -1815,7 +1815,7 @@ getcmdpos()			Number	return cursor position in command-line
 getcmdtype()			String	return current command-line type
 getcmdwintype()			String	return current command-line window type
 getcurpos()			List	position of the cursor
-getcwd()			String	the current working directory
+getcwd( [{scope}])			String	the current working directory
```

Comment:
@gregorias Is it? I don't know what the syntax rules for help files are, the original files are all over the place with tabs and spaces. For example, the line above has three tabs as well. This is one instance where I would prefer if it was just spaces instead of tabs all the way.